spec/features/users_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'rails_helper'
+
+RSpec.feature 'Users#index', type: :feature do
+  let(:user) { FactoryBot.create(:user) }
+
+  before do
+    FactoryBot.create(:post, user:)
+    FactoryBot.create_list(:user, 3)
+    visit users_path
+  end
+
+  scenario 'I can see the profile picture for each user' do
+    user.photo = '<span class="photo"> <img src="https://robohash.org/voluptatibusperspiciatissuscipit.png?size=300x300&amp;set=set1" width="100" height="100" /> </span>'
+    within(first('.user-list')) do
+      expect(page.body).to include(user.photo)
+    end
+  end
```

Comment:
- [ ] it would be better if you fix the failing tests in order to complete the requirements 
notice: in the screenshot, you are only running 5 tests  it should be 12 tests passing 
<img width="767" alt="Screenshot 2023-02-02 at 13 20 07" src="https://user-images.githubusercontent.com/61790587/216324007-d4421be6-19c3-484b-b144-f8c7274b813d.png">